### DISCUSSION TITLE
Autostart + Available from another IP (except localhost)

### DISCUSSION THREAD

**Discussion by LosPimpos** [flobz/psa_car_controller#638] [Q&A]
Hello,
is there a way to start the app automatically with the host?
I'm not too familiar with Linux... I would like to have the app start automatically with -R (5min).
Furthermore, I can only access the app from the host device via the localhost address. Is this also possible from a computer within the subnet? Ping works, but cannot access the website via port 5000.

Thanks and regards

**Comment by Drastef31** [comment]
Hello
To start the app from another IP address you can use the command line -l xx.xx.xx.xx to specify your IP
For me, command line is : psa-car-controller -c -r -d -R 5 -l 192.168.1.41
To autostart the app, I edited a .service file in /etc/systemd/system/ with the command

sudo nano /etc/systemd/system/psa-car-controller.service

Copy/paste the content

[Unit]
Description=PSA Controller
After=network-online.target

[Service]
ExecStart=/home/[user]/.local/bin/psa-car-controller -c -r -d -R 5 -l 192.168.1.41
WorkingDirectory=/home/[user]/
Restart=on-failure
RestartSec=1m
Environment=PYTHONUNBUFFERED=true
User=[user]

[Install]
WantedBy=multi-user.target  

and then use the command 

sudo systemctl enable psa-car-controller.service

to activate the service.

It works on debian on RPI. I do not know if it works on other distrib

**Reply by Matssa56** [reply]
Hey thanks for your help, do you need to replace [user] by the user that made the install? Or do you leave as is? Trying to make this work but still kinda tricky.

///EDIT
so yeah you need to adapt with your user and after a reboot it works fine.

**Reply by Drastef31** [reply]
👍
